sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTestCommon/CertificateBuilder.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Any idea why this file is flagged as a new file and not just a move?

Reply:
For this class (CertificateBuilder.cs) the certificatebuilder was implemented with [Mono.Security Nuget](https://github.com/Azure/azure-sdk-for-net/blob/2483720ad6ac1208e2044cd46490602353a9001b/src/SDKs/Batch/DataPlane/TestCommon/IntegrationTestCommon/CertificateBuilder.cs#L9) which doesn't work with .NetCore and has been unlisted from Nuget.org. Here I am generating the self signed certificate with BountyCastle libraries. I really need you to take a look at it to be sure it works at it should.

Reply:
Yes, I looked and this looked okay to me. Also, I ran tests which use this (in IntegrationTests) and they passed. So you're good here.